aaad/aaad.py

```diff
@@ -54,6 +58,14 @@ def do_GET(self):
 
         content_type = self.headers.getheader("content-type", "")
 
+        if auth_action == "filter_response":
+            filtered_response = self.filter_response(body, resource, act_as_user, permissions)
+            self.send_response(200)
```

Comment:
When it enters the filterResponse part, it makes sure that the call flow hasn't broken earlier.